public/assets/js/admin/adminFormArticle.js

```diff
@@ -0,0 +1,31 @@
+let buttonValidate = document.getElementById("validate");
+let buttonValidateForm = document.getElementById("validateForm");
+let buttonCancel = document.getElementById("cancel");
+let buttonCancelForm= document.getElementById("cancelForm");
+let modalText = document.getElementById("modalText");
+
+let modal = document.getElementById("modal");
+
+buttonValidate.addEventListener("click", () =>{
+    modal.style.display = "block";
+    modalText.textContent = "Désirez-vous enregistrer votre article?";
+})
```

Comment:
instead of changing css style directly in JS, prefer toggle or add/remove classes which will have the css properties